staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -265,6 +301,16 @@ func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope *RequestSc
 			scope.err(err, w, req)
 			return
 		}
+
+		if options != nil && ptr.Deref(options.IgnoreStoreReadErrorWithClusterBreakingPotential, true) {
```

Comment:
needs to be featuregated.  Also after we have deleteoptions, if the gate is off, need to set the `IgnoreStoreReadErrorWithClusterBreakingPotential` to nil.